src/components/PostForm/components/SelectField/SelectField.spec.tsx

```diff
@@ -0,0 +1,90 @@
+import { fireEvent, screen, render } from '@testing-library/react'
+
+import { renderWithProviders } from '../../../../tests/lib/helpers'
+import { PostFieldProps } from '../../FieldRenderer'
+
+import { SelectField } from './SelectField'
+
+describe('SelectField', () => {
+  const fieldProps: Omit<PostFieldProps, 'onChange' | 'onBlur'> = {
+    field: {
+      key: 'select',
+      title: 'SelectField',
+      type: 'string',
+      directive: 'select',
+      minimum: 0,
+      maximum: 1,
+      options: [
+        { name: 'Approach', value: 'Approach Value' },
+        { name: 'Pre-Gospel', value: 'Pre-Gospel Value' },
+        { name: 'Gospel', value: 'Gospel Value' }
+      ]
+    },
+    value: ''
+  }
+
+  it('should display proper values for single select', async () => {
+    renderWithProviders(<SelectField {...fieldProps} />)
+
+    const selectField = await screen.findByRole('button')
+    expect(selectField).toBeInTheDocument()
+    fireEvent.mouseDown(selectField)
+
+    const option = await screen.findByRole('option', { name: 'Approach' })
+    expect(option).toBeInTheDocument()
+
+    fireEvent.click(option)
+    expect(
+      await screen.findByRole('button', { name: 'Approach' })
+    ).toBeInTheDocument()
+  })
+
+  it('should toggle on clicking twice for multiple select', async () => {
```

Comment:
could you please make this more descriptive? I'm guessing you mean toggle on chip for multiple select?